src/apps/front-office/account/pages/register-page/RegisterPage.tsx

```diff
@@ -0,0 +1,118 @@
+import { trans } from "@mongez/localization";
+import { Form } from "@mongez/react-form";
+import Helmet from "@mongez/react-helmet";
+import { Link, navigateBack } from "@mongez/react-router";
+import { BookOpen } from "lucide-react";
+import Footer from "src/apps/front-office/design-system/layouts/Footer/Footer";
+import Header from "src/apps/front-office/design-system/layouts/Header/Header";
+import endpoint from "src/shared/endpoint";
+//import { useCreateAccount } from "../../hooks";
+import CheckboxInput from "./components/CheckboxInput";
+import EmailInput from "./components/EmailInput";
+import HeadOfRegister from "./components/HeadOfRegister";
+import PasswordInput from "./components/PasswordInput";
+import RadioGroup from "./components/RadioGroup";
+import RadioInput from "./components/RadioInput";
+import SocialLogin from "./components/social-login/SocialLogin";
+import TextInput from "./components/TextInput";
+import "./input.css";
+import "./locales";
+
+export default function RegisterPage() {
+  //const createAccount = useCreateAccount();
+  const submitForm = async ({ values }) => {
+    try {
+      await endpoint.post("/register", values);
+      navigateBack();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  // const submitForm=({values,form})=>{
+  //   createAccount({values,form})
+  // }
+  return (
+    <>
+      <Helmet
+        title={trans("registerTitle")}
+        appendAppName={false}
+        keywords={["online", "store", "متجر"]}
+        description={trans("registerDescription")}
+      />
+      <Header />
+      <div className="container lg:w-5/6 xl:w-4/6 grid gap-7 grid-cols-4 my-20">
+        <div className="col-span-4 md:col-span-3 lg:col-span-2 ms-3 lg:ms-0">
+          <HeadOfRegister />
+          <Form onSubmit={submitForm} className="flex flex-col gap-2">
+            <TextInput
+              type="text"
+              name="userName"
+              placeholder={trans("namePlaceHolder")}
+              required
+            />
+            <EmailInput
+              type="email"
+              name="email"
+              placeholder={trans("emailPlaceHolder")}
+              required
+            />
+            <PasswordInput
+              type="password"
+              name="password"
+              placeholder={trans("passwordPlaceHolder")}
+              minLength={8}
+              maxLength={16}
```

Comment:
Why would we strict the password with max length, specially with 16, my own password is way longer than 16 😃